README.md

```diff
@@ -480,6 +480,9 @@ this produces the output:
 ```python
 [[-0.52324814]]
 ```
+### Using SPMN module
+
+* See [SPMN](https://github.com/c0derzer0/SPMN) for sample datasets and instructions for using SPMN module
```

Comment:
If possible, please attach a set of instructions to the base of this library as well about a sample test case and how to run it + expected output

Reply:
I have a sample test case, running instructions and output in the link given. I can put that in here.